src/util/apiUtils.js

```diff
@@ -69,6 +69,16 @@ export const parseApiValue = ([response, body]) => {
 		return formatApiError(new Error(response.statusMessage));
 	}
 	try {
+		if (response.statusCode === 204) { //NoContent response type
+			return {
+				value: {},
+				meta: {
+					success: true,
```

Comment:
is this useful? unless there's a `value.error`, we should be able to assume it's a success response